Header header
int32 velocity
float32 angle
bool isEmerge
